circle.yml

```diff
@@ -6,7 +6,9 @@ machine:
 
 dependencies:
   override:
+    - brew install swiftlint
     - bundle install
+    - rake danger
```

Comment:
Danger should look at runtime, via the Circle API to infer whether circle thinks its a PR